reactioncommerce/reaction
# Don't add a prefix to Paypal return URL when there is a single shop

The Paypal plugin was adding the shop slug to the return url even when we only had one shop, where we shouldn't be using the shop prefix. Changed the behavior of `getShopPrefix` slightly but tested the other places it is used and seems to be fine.

### Steps to test
1. Place an order via Paypal Express with default data
1. Use a `Shops.json` with only one store
1. Place an order via Paypal Express and confirm that it completes.
